contentful.json

```diff
@@ -57,23 +57,23 @@
     "Client Side Forms": "No",
     "Plugin System": "No",
     "Customizable UI": "Yes",
-    "Custom Field Types": "No",
+    "Custom Field Types": "Yes",
     "User Management": "Yes",
     "Role Based Permissions": "Yes",
     "Document Level Permissions": "No",
-    "OAuth 2.0 Support": "No",
-    "Auditing": "No",
+    "OAuth 2.0 Support": "Yes",
+    "Auditing": "Yes",
```

Comment:
Can be achieved through the use of webhooks and 3rd party logging service
https://www.contentful.com/developers/docs/concepts/webhooks/
https://www.contentful.com/developers/docs/references/content-management-api/#/reference/webhooks

Reply:
@Jotschi any thoughts about this? Contentful's approach is to make it easy to integrate with the logging/auditing platform a company is already using, rather than have one more portal/place to look at for auditing info

Reply:
I'm hesitant to accept third party integrations which use use the regular API of the CMS. It is hard to trace the support and lifecycle of those and the customer experience is often not ideal. I assume the auditing UI is not part of contentful UI? Does the webhook expose _who_ edited something?

Reply:
Yes there is a relatively new webhook transformations function which allows you to change the structure of the webhook body to match the receiving side's expected structure. Included with webhook transformations is the ability to include user info in the webhook body.